_source/_posts/2020-01-21-angular-material-login.md

```diff
@@ -194,16 +188,9 @@ Start with the main application component. Open `src/app/app.component.html` and
 ```
 {% endraw %}
 
-This code contains the main toolbar and menu of the application. Only a small amount of styling is needed. Paste the following into `src/app/app.component.css`.
-
-```css
-.expanded-toolbar {
-  justify-content: space-between;
-  align-items: center;
-}
-```
+This code contains the main toolbar and menu of the application. Because we used the Flex-Layout directives to set the layout for the elements, there's no extra styles needed.
```

Comment:
```suggestion
This code contains the main toolbar and menu of the application. Because we used the Flex-Layout directives to set the layout for the elements, no extra styles are needed.
```
"are" not "is"